### ISSUE TITLE
idea: Command palette for executing shortcuts

### ISSUE THREAD

**Issue by Unstuffedspare** [janhq/jan#3767]
### Problem Statement

I always dislike navigating apps with my mouse - as a 120wpm typist, the more keyboard actions available the better!!!!

### Feature Idea

I would love to see Jan build in something similar to Obsidian's/Superhuman's command palettes.

This would allow for quicker navigation of the app, especially when executing more complex actions like tweaking settings, where you need to edit a value. With a command system, the user could  type the settings name and then enter a value. It would also make shortcuts accessible for those who haven't yet learned the full keyboard list.

Keyboard shortcut legends should be displayed besides applicable actions names, helping reinforce them into the users memory.

We really need to be able to set our own binds, but as a starting point, I would recommend setting the shortcut to Ctrl/Cmd-K.

![image](https://github.com/user-attachments/assets/808d6c4f-ac0c-4bd3-8f42-64067976a788)
![image](https://github.com/user-attachments/assets/8e894dbd-f6b4-4785-ba20-cbaa270a707b)

**Comment by imtuyethan** [comment]
Related: 
https://github.com/janhq/jan/issues/2757
https://github.com/janhq/jan/issues/3587

Original epic ticket was archived: https://github.com/janhq/jan/issues/3555 and moved to Discussion: https://github.com/orgs/janhq/discussions/3572

I think we should have a big epic item to track this?

**Comment by Worgle123** [comment]
Hey there - Unstuffedspare here.

I finally managed to unlock my main GitHub, so will be switching back to it for most uses. Thanks!!

**Comment by imtuyethan** [comment]
Closed as a sub-task of https://github.com/janhq/jan/issues/4255
